src/services/gitLinks.jsx

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Lots of extra whitespace and you've got some unnecessary objects in your fetches as well.